Added desktop version

ahmadsarfrazx opened the pull request:
- Added a new branch named desktop-version
- Added a new file named query.css
- Add media queries in query.css file to make desktop portfolio responsive also identical to Figma
- Removed the width from the .container element as per the requirement
- Reduced the white space from project description and cards
- Removed the bottom line from the upside of the framework item
- Made all required changes
- Adjusted border radius as per requirement which matches the Figma template
- Implemented the dropdown icons as suggested by the previous reviewer
- All required changes have been implemented successfully and the project is complete now
- Removed list style in the about section
- Changed language icon from column to row

ahmadsarfrazx committed:
Added desktop version

bienvenuushindi commented on query.css:
```diff
@@ -0,0 +1,356 @@
+  /* end card */
+
+  /* about */
+  .about {
```
:red_circle: Kindly notice the top-right border radius on this section. Please make sure you add the radius to match the Figma design

Your Design
![border](https://user-images.githubusercontent.com/26736582/200088635-280464fe-d0d0-4a19-b482-8cf4a29da58b.png)

Figma Design
![borderfigma](https://user-images.githubusercontent.com/26736582/200088655-cfc12ddd-8070-4cfc-b52b-6c619cd816fc.png)

Debas-31 replied:
- [ ] Make sure this issue is addressed with the feedback from the previous reviewer about the border radius for the top right corner based on the Figma template design; please fix this issue

bienvenuushindi commented on query.css:
```diff
@@ -0,0 +1,356 @@
+    margin-left: 200px;
+  }
+
+  .about_ul {
```
:red_circle: Kindly notice that each language item has an icon at the left. Your design has no icon for this section as you can in this screenshot

Your Design
![yrnoicon](https://user-images.githubusercontent.com/26736582/200089031-3c0c4e9b-692d-4b6c-8b20-a7d09e973a6a.png)

Figma Design
![noarrows](https://user-images.githubusercontent.com/26736582/200089045-be715ea0-9ae9-4baa-a19e-b3bb70608aea.png)

:red_circle: Please make sure you include the icons as shown on this Figma design :arrow_up_small:

bienvenuushindi replied:
:red_circle: Kindly notice that the bottom line is only applied to the last two items (Frameworks & Skills). Please make sure you remove the bottom line on the first item (language)

Your Design
![yrborderless](https://user-images.githubusercontent.com/26736582/200089456-013f0d6f-de18-459e-91b1-6c2c0cc43710.png)

Figma Design
![figborderless](https://user-images.githubusercontent.com/26736582/200089470-243f69ba-1d3c-4143-88d9-2489449fd4f5.png)

Debas-31 replied:
- [ ] Please implement icons in the above section, asked to work by the previous reviewer :+1:

bienvenuushindi commented on query.css:
```diff
@@ -0,0 +1,356 @@
+    margin-left: 200px;
+  }
+
+  .desktop-card,
```
:red_circle: Kindly notice the white space after the project description on odd cards and before on even cards. Kindly reduce the white space to match the Figma design

Your Design
![yrextraspace](https://user-images.githubusercontent.com/26736582/200090122-d71d33de-8c7c-43b4-a68d-5d2fb8f09dde.png)

Figma Design

![extraspace](https://user-images.githubusercontent.com/26736582/200090143-7b49a5a5-8b0f-4ea8-83e8-c4edc81da9ce.png)

bienvenuushindi reviewed (changes requested):
### Hi @ahmadsarfrazx ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- [ ] N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

bienvenuushindi commented on query.css:
```diff
@@ -0,0 +1,356 @@
+  .end-line {
+    display: flex;
+  }
+}
```
:red_circle: Kindly make sure you remove the fixed width applied on the` .container` element as it makes your design unresponsive when resizing your screen. You can see its effect in the screenshot below :arrow_down: 
![chnage](https://user-images.githubusercontent.com/26736582/200091286-3c8fe5c4-f5b6-47a2-aaab-92e0b3d45b49.gif)

ahmadsarfrazx committed:
Made required changes

ahmadsarfrazx commented:
- Removed the width from the .container element as per the requirement
- Reduced the white space from project description and cards
- Removed the bottom line from the upside of the framework item
- Made all required changes

Debas-31 reviewed (changes requested):
> ## Hi @ahmadsarfrazx :wave:, 
# Status: Require Change :recycle:
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<img src="https://media4.giphy.com/media/nYYtQdLqwwlTe3ccKh/giphy.gif"/>

 ## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- Commit message :heavy_check_mark:
 ## Required Changes ♻️
_Check the comments above the review._

## Optional suggestions
- No 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ahmadsarfrazx committed:
Did the required changes

ahmadsarfrazx commented:
- Adjusted border radius as per requirement which matches the Figma template
- Implemented the dropdown icons as suggested by the previous reviewer
- All required changes have been implemented successfully and the project is complete now

Olamarx commented on index.html:
```diff
@@ -163,74 +164,95 @@ <h2 class="title">Multi-Post Stories</h2>
+          </div>
+          <ul class="language-list">
+            <li>
+              <img src="assets/javascript.svg" alt="JavaScript" />
+              JavaScript
+            </li>
+            <li>
+              <img src="assets/html.svg" alt="HTML" />
+              HTML
+            </li>
+            <li>
+              <img src="assets/css.svg" alt="CSS" />
+              CSS
+            </li>
```
- [ ] While you were implementing the changes, you forgot to remove the list style(dots beside the languages, framework and skills) also, Javascript, html and CSS are in rows, not column.
### Your webpage
![image](https://user-images.githubusercontent.com/86200356/200123393-a0d1a7b2-072b-4b43-bcf1-274c27cf8107.png)
### Figma
![image](https://user-images.githubusercontent.com/86200356/200123451-6bed23aa-e26c-4686-a350-e153becfa018.png)

Olamarx reviewed (changes requested):
Hi @ahmadsarfrazx, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<img src="https://media1.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif"/>

 ## Highlights
- All linters passed ✔️ 
- Good commit messages 👍🏼 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ahmadsarfrazx committed:
Done with required changes

ahmadsarfrazx commented:
- Removed list style in the about section
- Changed language icon from column to row

zieeco reviewed (approved):
# status: APPROVED :green_circle: :100: 

Hi, @ahmadsarfrazx , 


![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966](https://user-images.githubusercontent.com/85113567/164945882-0a9554ea-956c-4fa0-a95c-379285283c1d.gif)


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
